src/main/java/jR_Project3/servlets/PageServlet.java

```diff
@@ -0,0 +1,52 @@
+package jR_Project3.servlets;
+
+import jR_Project3.dto.InfoFormDTO;
+import jR_Project3.models.CookiesNames;
+import jR_Project3.models.books.Book;
+import jR_Project3.models.books.Part;
+import jR_Project3.services.BookPartsService;
+import jR_Project3.services.StepsInGameService;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.*;
+import java.io.IOException;
+
+import static jR_Project3.models.ReqParameters.PART;
+import static jR_Project3.models.SessionAttributes.*;
+
+@WebServlet("/page")
+@Slf4j
+public class PageServlet  extends HttpServlet {
+    //TODO Нужно ли объявлять экземпляры тут? Или лучше внутри методов? Или Все таки методы BookPartsService делать статическими?
+    private final BookPartsService bookPartsService = new BookPartsService();
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        log.info("started");
```

Comment:
add more info to the log
info about the request, http method, endpoint, unique ids etc.